arwen/helpers.go

```diff
@@ -104,6 +104,11 @@ func SetLoggingForTests() {
 	logger.ToggleLoggerName(true)
 }
 
+// DisableLoggingForTests sets log level to *:NONE
+func DisableLoggingForTests() {
+	_ = logger.SetLogLevel("*:NONE")
```

Comment:
I would go one step further and let mandos also specify the log level "NONE", "INFO", "DEBUG" ... I think

Reply:
This should be easy to use. Logs = true or false is what we need.